server/src/main/java/org/jboss/as/server/deployment/annotation/ResourceRootIndexer.java

```diff
@@ -62,7 +62,7 @@ public static void indexResourceRoot(final ResourceRoot resourceRoot) throws Dep
                 ServerLogger.DEPLOYMENT_LOGGER.tracef("Found and read index at: %s", indexFile);
                 return;
             } catch (Exception e) {
-                ServerLogger.DEPLOYMENT_LOGGER.cannotLoadAnnotationIndex(indexFile.getPathName());
+                ServerLogger.DEPLOYMENT_LOGGER.cannotLoadAnnotationIndex(indexFile.getPathName(), e.toString());
```

Comment:
Should it be `e.getMessage()`?

Reply:
The e.toString() is more complete. We could have the exception without any message. The to show the exception message if it exist plus the exception class name.